### PR TITLE
enhancement(builds): Adding commit messages to Vela build list

### PR DESCRIPTION
**Pull request by jenmartTarget**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ services:
 
   server:
     container_name: server
-    image: target/vela-server
+    image: localhost:5000/vela-server
```
Comment (davidvader) [inline docker-compose.yml:21]: we should revert the changes made to the default docker-compose configurations

File: src/elm/Main.elm [hunk 2]
```diff
@@ -1588,7 +1588,7 @@ viewContent model =
             )
 
         Pages.AddRepositories ->
-            ( "Add Repositories"
+            ( "Add Rrepositories"
```
Comment (davidvader) [inline src/elm/Main.elm:1591]: might want to revert this 😅

File: src/elm/Pages/Home.elm [hunk 3]
```diff
@@ -68,8 +68,8 @@ view user filter { toggleFavorite, search } =
                     [ text "To display a repository here, click the "
                     , SvgBuilder.star False
                     ]
-                , p [] [ text "Add repositories from your GitHub account to Vela now!" ]
-                , a [ class "button", Routes.href Routes.AddRepositories ] [ text "Add Repositories" ]
+                , p [] [ text "Add repositoriees from your GitHub account to Vela now!" ]
```
Comment (davidvader) [inline src/elm/Pages/Home.elm:71]: same here

File: src/scss/_main.scss [hunk 4]
```diff
@@ -443,6 +442,14 @@ nav {
   margin: 0 8px 0 8px;
 }
 
+.row .message {
+  font-size: 16px;
```
Comment (davidvader) [inline src/scss/_main.scss:446]: kinda picky, but we're trying to use `rem` rather than `px` for font-size (although some of the older style is still in px and needs to be updated) but good to get in the habit
I think you can satisfy
```font-size: 16px;```
with
```font-size: 1rem;```

Comment (davidvader) [reply src/scss/_main.scss:446]: more info: https://engageinteractive.co.uk/blog/em-vs-rem-vs-px

File: docker-compose.yml (hunk 1 above)
Comment (jenmartTarget) [reply docker-compose.yml]: Fixed.

File: src/elm/Main.elm (hunk 2 above)
Comment (jenmartTarget) [reply src/elm/Main.elm]: Fixed.

File: src/scss/_main.scss (hunk 4 above)
Comment (jenmartTarget) [reply src/scss/_main.scss]: Fixed. Should I make this conversion to the other uses of px?

File: docker-compose.yml [hunk 5]
```diff
@@ -33,7 +33,7 @@ services:
       VELA_PORT: ':8080'
       VELA_SECRET: zB7mrKDTZqNeNTD8z47yG4DHywspAh
       VELA_SOURCE_DRIVER: github
-      VELA_SOURCE_URL: https://github.com/
+      VELA_SOURCE_URL: https://git.target.com
```
Comment (davidvader) [inline docker-compose.yml:36]: this should be reverted :)

File: src/elm/Pages/Home.elm [hunk 6]
```diff
@@ -69,7 +69,7 @@ view user filter { toggleFavorite, search } =
                     , SvgBuilder.star False
                     ]
                 , p [] [ text "Add repositories from your GitHub account to Vela now!" ]
-                , a [ class "button", Routes.href Routes.AddRepositories ] [ text "Add Repositories" ]
+                , a [ class "button", Routes.href Routes.AddRepositories ] [ text "Add Repositoriess" ]
```
Comment (davidvader) [inline src/elm/Pages/Home.elm:72]: should this be reverted as well? :)

File: docker-compose.yml (hunk 5 above)
Comment (jenmartTarget) [reply docker-compose.yml]: Fixed.

File: src/elm/Pages/Home.elm (hunk 6 above)
Comment (jenmartTarget) [reply src/elm/Pages/Home.elm]: Fixed.

**Comment by davidvader** [comment]
redacting my review because I forgot to mention adding a test or two for the new content we're adding to the page :D
